esprima.js

```diff
@@ -4696,12 +4804,20 @@
                 token = lookahead;
                 isStatic = false;
                 computed = match('[');
-                key = parseObjectPropertyKey();
-                if (key.name === 'static' && lookaheadPropertyName()) {
-                    token = lookahead;
-                    isStatic = true;
-                    computed = match('[');
+                if (match('*')) {
+                    lex();
```

Comment:
This should be taken care by `tryParseMethodDefinition`. I agree that it would be nice to centralize the handling`*` but this will do for now.